app/javascript/crayons/Modal/Modal.jsx

```diff
@@ -25,34 +40,19 @@ export const Modal = ({
   overlay,
   onClose,
 }) => {
-  const Icon = () => (
-    <svg
-      width="24"
-      height="24"
-      viewBox="0 0 24 24"
-      className="crayons-icon"
-      xmlns="http://www.w3.org/2000/svg"
-      role="img"
-      aria-labelledby="714d29e78a3867c79b07f310e075e824"
-    >
-      <title id="714d29e78a3867c79b07f310e075e824">Close</title>
-      <path d="M12 10.586l4.95-4.95 1.414 1.414-4.95 4.95 4.95 4.95-1.414 1.414-4.95-4.95-4.95 4.95-1.414-1.414 4.95-4.95-4.95-4.95L7.05 5.636l4.95 4.95z" />
-    </svg>
-  );
-
   return (
     <div
       className={`crayons-modal${getAdditionalClassNames({
         size,
         className,
       })}`}
     >
-      <div className="crayons-modal__box">
+      <dialog open className="crayons-modal__box">
```

Comment:
Using the [\<dialog /\>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) HTML element. `open` is always present because of the way we currently render modals. If this component is rendered, it means the modal is open.